### ISSUE TITLE
SumatraPDF crashes when zooming and/or scrolling in CHM file

### ISSUE THREAD

**Issue by Tinase-nau** [sumatrapdfreader/sumatrapdf#4480]
**SumatraPDF version**
3.5.2

**Describe the bug**
SumatraPDF crashes a few seconds after scrolling or zooming in 3x and then scrolling in any CHM.

**To Reproduce**
1. Get a random test CHM, for e.g. get the official AutoIt.chm
2. Open it in SumatraPDF
3. Open up a scrollable page
4. Scroll, then wait a bit - SumatraPDF will crash exit. IF crash didn't occur, zoom the page in 3x. then try scrolliing - the scrolling will not function anymore, or SumatraPDF will crash.

**Expected behavior**
Not crash when scrolling or zooming in CHM file.

**File that reproduces the problem**
All CHM files? Tried with a few, e.g. AutoIt.chm.

**Comment by kjk** [comment]
Does it happen in pre-release (https://www.sumatrapdfreader.org/prerelease) ?

**Comment by Tinase-nau** [comment]
I tried AutoIt.chm in pre:
When SumatraPDF is in default sizing and I open the CHM and then maximise the window, it doesn't resize the CHM, the right side just shows corruption.
When I open the CHM in maximized window then all is okay.
Then began bug with zooming: SumatraPDF doesn't show the CHM in normal size anymore, either it's tiny or mega big.
I remembered that it is possible to reset view with Ctrl+0, that worked. But the zooming in and zooming out is still broken.
